pom.xml

```diff
@@ -79,6 +79,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <compiler.version>8</compiler.version>
+        <revision>9.1.0-SNAPSHOT</revision>
```

Comment:
This has to be manually changed to 9.1.0 for release and tag, and then 9.2.0-SNAPSHOT for the next dev stage. 
What do you mean as same as before? You are not an upstream release manager, I don't think you have gone through all Java agent release process.

Reply:
> This has to be manually changed to 9.1.0 for release and tag, and then 9.2.0-SNAPSHOT for the next dev stage.

This doesn't actually need to be manually changed, when you run the release command, you'll be prompted like before about what the version is, what the next SNAPSHOT version is